aws/cloudformation/lambdas/marketing-router/index.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const { MARKETING_DOMAIN } = process.env;
+
+const marketingPaths = {
+  // Add key-value pairs for each path that should be served by the CMS
+  // e.g. '/videos': true,
+}
+
+export const handler = (event, context, callback) => {
+  const request = event.Records[0].cf.request;
```

Comment:
It'd be nice to have some defensive coding in case any of those keys are undefined to simply pass the request down. I believe node 22 supports optional chaining so we can use that to defensively get the request, and if unavailable, pass down